internal/server/services.go

```diff
@@ -54,7 +54,9 @@ func InitServices(deps ServiceDeps) (*Services, error) {
 		return nil, err
 	}
 
-	auditRepository := audit_repos.NewPostgresRepository(store.DB())
+	sqldb, _ := store.DB().DB()
+
+	auditRepository := audit_repos.NewPostgresRepository(sqldb)
```

Comment:
we also need to make sure the audit's migration is being called from our migration